lib/modules/platform/azure/index.ts

```diff
@@ -301,11 +301,15 @@ export async function findPr({
     const prs = await getPrList();
 
     prsFiltered = prs.filter(
-      (item) => item.sourceRefName === getNewBranchName(branchName)
+      (item) =>
+        item.sourceRefName?.toLowerCase() ===
+        getNewBranchName(branchName)?.toLowerCase()
     );
 
     if (prTitle) {
-      prsFiltered = prsFiltered.filter((item) => item.title === prTitle);
+      prsFiltered = prsFiltered.filter(
+        (item) => item.title.toLowerCase() === prTitle.toLowerCase()
```

Comment:
use `toUpperCase` as there are chars which doesn't have a lower case equivalent

Reply:
No problem. Out of curiosity: What happens with the characters? The languages I know just keep them untouched, which would make the behaviour equivalent.

Reply:
I first saw a lint rule on dotnet, which suggested to use upper case


https://stackoverflow.com/a/14128850